test/integration/server-discovery-and-monitoring/server_discovery_and_monitoring.prose.test.ts

```diff
@@ -110,13 +110,15 @@ describe('Server Discovery and Monitoring Prose Tests', function () {
     */
 
     let client: MongoClient;
+    let utilClient: MongoClient;
     const events: string[] = [];
     beforeEach(async function () {
       client = this.configuration.newClient({
         directConnection: true,
         appName: 'SDAMPoolManagementTest',
-        heartbeatFrequencyMS: 500
+        heartbeatFrequencyMS: 100
       });
+      utilClient = this.configuration.newClient({ directConnection: true });
```

Comment:
Our general pattern in tests is to have a util client for setting fail points, so I changed this in this test.